govcd/system.go

```diff
@@ -500,3 +500,16 @@ func QueryProviderVdcByName(vcdCli *VCDClient, name string) ([]*types.QueryResul
 
 	return results.Results.VMWProviderVdcRecord, nil
 }
+
+// QueryOrgVdcNetworkByName finds a org VDC network by name which has edge gateway as reference
+func QueryOrgVdcNetworkByName(vcdCli *VCDClient, name string) ([]*types.QueryResultOrgVdcNetworkRecordType, error) {
```

Comment:
As the usage of this function is not visible from this PR, I'm just curious, why do we need this new function when we have remotely similar `func (vdc *Vdc) FindVDCNetwork(network string) (OrgVDCNetwork, error)` in `vdc.go`? Another way of asking, why do we need to query as opposed to finding in VDC?

Reply:
This function was result of my other POC where I need figure out network and edge gtw relationship. Find and Query return different set of data about network. I decided to add this function for future use cause already created it, didn't want to delete.